tests/integ/scala/test_dataframe_suite.py

```diff
@@ -980,6 +981,86 @@ def test_select(session):
     # select(Seq[Column]) with col("a") + col("b")
     assert df.select([col("b"), col("a") + col("c")]).collect() == expected_result
 
+    # test single output column udtf
+    class TwoXUDTF:
```

Comment:
Probably we should make a new test function for your changes. Ideally it should be put in tests/integ/test_dataframe.py. `scala/...suite.py` is especially for tests ported from scala snowpark (though we have many violations in the code)